packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -83,13 +85,19 @@ export interface ISiteState {
     formView: boolean;
     devToolsView: boolean;
     locale: string;
+    theme: IThemeValues;
 }
 
 export enum SiteSlot {
     category = "category",
     categoryIcon = "category-icon"
 }
 
+export interface IThemeValues {
+    name: string;
```

Comment:
seems like we are using name like a uuid, a better pattern might be:
```
{
    uuid: string;
    displayName: string;
    background: string;
}
```

I would prefer a separation of concerns here where the eventual value of the selected option is different from the string being displayed.

Reply:
Rather than doing this eventually, can we do this in this pr? Do we want uuid or just ID. If a user is passing, I think id works better from a clarity standpoint.

Do we want to hand them back the id in this case then rather than the display name? Seems like a better option.

Reply:
I think we want to pass them the id as it should be unique.